code/style.css

```diff
@@ -2,6 +2,335 @@
+  margin: 0;
+}
+
+.button-primary {
+  margin-top: 20px;
+  background: #f36127;
+  border-radius: 100px;
+  padding: 12px 32px;
+  border: none;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  cursor: pointer;
+
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h1, h2, h3 {
+  font-family: 'Bebas Neue';
+  font-style: normal;
+  margin-top: 0;
+}
+
+h1 {
+  font-size: 42px;
+  line-height: 55px;
+}
+
+h2 {
+  font-size: 30px;
+  line-height: 40px;
+  margin-bottom: 16px;
+}
+
+
+.placementcard > h2  {
+  margin: 0 auto 16px 16px;
+}
+
+
+h3 {
+  font-size: 24px;
+  line-height: 32px;
+}
+
+p {
+  font-family: 'Bebas Neue';
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  
+}
+
+label {
+  margin: 0 0 8px 16px;
+}
+
+.orangetext {
+  color: #f36127;
+
 }
+.formquestion {
+  font-family: 'Bebas Neue';
+  font-weight: 700;
+  font-size: 18px;
+}
+
+.p-check {
+  font-family: 'Bebas Neue';
+  font-size: 14px;
+}
+
+input[type=text], input[type=email] {
+  border-radius: 24px;
+  border: 1px solid rgba(0,0,0,0.20);
+  padding: 10px 24px;
+  font-family: 'Arial'; 
+  font-size: 14px;
+  line-height: 24px;
+  margin-bottom: 24px;
+}
+
+input[type=text]:focus, input[type=email]:focus {
+  border: 0,5px solid #f36127;
+  outline-width: 0;
+  box-sizing: border-box;
+  box-shadow: inset 0 0 0px 2px #f36127;
+}
+
+.form-checkcontainer, .form-checkcontainer {
+  width: 100%;
+  padding: 0 16px 0 16px;
+  margin-bottom: 24px;
+}
+
+.checkgroup {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+}
+
+.checkbox {
+  appearance: none;
+  border: 1px solid rgba(0,0,0,0.3);
+  width: 18px;
+  height: 18px;
+  margin: 0px;
+  border-radius: 4px;
+  outline-width: 0;
+}
+
+.checkbox:checked {
+  background: #f58822;
+  border: 1px solid #f36127; 
+  box-shadow: inset 0px 0px 0px 2px #FFF;
+  outline-width: 0;
+}
+
+.placementcard {
+margin: 20px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+}
+
+main {
+  max-width: 100%;
+  margin: 0 auto;
+  padding: 40px;
+  display: grid;
+  align-items: start;
+  grid-gap: 40px 20px;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.card {
+  background: #FFFFFF;
+  box-shadow: 0px 24px 40px rgba(0, 0, 0, 0.4);
+  border-radius: 4px;
+  color: rgba(0,0,0,0.87);
+  padding: 24px;
+  height: 500px;
+  width: 100%;
+  height: auto;
+}
+
+/** Accordion code here **/ /**https://www.w3schools.com/howto/howto_js_accordion.asp**/
+
+.card-accordion {
+  background: #FFFFFF;
+  box-shadow: 0px 24px 40px rgba(0, 0, 0, 0.4);
+  border-radius: 4px;
+}
+
+/** Mellanrum i frågeformuläret**/
+
+.accordion-item {
+  background-color: white;
+  color: #111;
+  margin: 1rem 0;
+}
+
+.accordion-item-header {
+  padding: 8px 40px 8px 24px;
+  min-height: 3.5rem;
+  line-height: 1.25rem;
+  display: flex;
+  align-items: center;
+  position: relative;
+  cursor: pointer;    /**https://developer.mozilla.org/en-US/docs/Web/CSS/cursor**/
```

Comment:
I see you wrote a link to the source where you found a solution, that is smart! :) next time when you have the same problem, you can go back and look at a link. Super!